fxapom/pages/base.py

```diff
@@ -14,7 +14,7 @@ def __init__(self, selenium, timeout=TIMEOUT):
         self._main_window_handle = self.selenium.current_window_handle
 
     def switch_to_main_window(self):
-        self.selenium.switch_to_window(self._main_window_handle)
+        self.selenium.switch_to.window 
```

Comment:
Looks like there is some white space here at the end. 

Also, this isn't currently switching to a window. The ```window``` is a function and still needs to know what window to switch to. See this [example](https://github.com/mozilla/FoxPuppet/blob/d1e47afc3c364ebf3c34a43aaa45f95d40ff51dc/foxpuppet/windows/base.py#L55)

Reply:
Sorry for these problems and thanks for the review